### PR TITLE
Tests fail to compile when the contract contains a long value

### PR DESCRIPTION
**Pull request by FelipeAdorno**
without this change when a contract contains a long value inside the body, then the code fails to compile since the provided number is assumed to be an integer.
with this change we're appending an `L` literal to that value for the compiler to assume that it's a long value.

### CODE REVIEW

**Comment by pivotal-issuemaster** [comment]
@FelipeAdorno Please sign the [Contributor License Agreement](https://cla.pivotal.io/sign/spring?repositoryId=spring-cloud/spring-cloud-contract&pullRequestId=358)!

[Click here](https://cla.pivotal.io/sync/spring?repositoryId=spring-cloud/spring-cloud-contract&pullRequestId=358) to manually synchronize the status of this Pull Request.

See the [FAQ](https://cla.pivotal.io/about) for frequently asked questions.

**Comment by pivotal-issuemaster** [comment]
@FelipeAdorno Thank you for signing the [Contributor License Agreement](https://cla.pivotal.io/sign/spring?repositoryId=spring-cloud/spring-cloud-contract&pullRequestId=358)!

File: spring-cloud-contract-verifier/src/test/groovy/org/springframework/cloud/contract/verifier/builder/MethodBodyBuilderSpec.groovy [hunk 1]
```diff
@@ -281,4 +281,36 @@ DocumentContext parsedJson = JsonPath.parse(json);
...
+			blockBuilder.toString().contains("""assertThatJson(parsedJson).field("['createdAt']").isEqualTo(1502766000000L)""")
+			blockBuilder.toString().contains("""assertThatJson(parsedJson).field("['updatedAt']").isEqualTo(1499476115000L)""")
+		and:
+			SyntaxChecker.tryToCompileWithoutCompileStatic(methodBuilderName, blockBuilder.toString())
+		and:
+			stubMappingIsValidWireMockStub(contractDsl)
+		where:
+			methodBuilderName           | methodBuilder
+			"MockMvcSpockMethodBuilder" | { Contract dsl -> new MockMvcSpockMethodRequestProcessingBodyBuilder(dsl, properties) }
+			"MockMvcJUnitMethodBuilder" | { Contract dsl -> new MockMvcJUnitMethodBodyBuilder(dsl, properties) }
```
Comment (marcingrzejszczak) [inline spring-cloud-contract-verifier/src/test/groovy/org/springframework/cloud/contract/verifier/builder/MethodBodyBuilderSpec.groovy]: I think we're missing `JaxRsClientSpockMethodRequestProcessingBodyBuilder` and `JaxRsClientJUnitMethodBodyBuilder` in the list here, aren't we?

File: spring-cloud-contract-verifier/src/main/groovy/org/springframework/cloud/contract/verifier/util/DelegatingJsonVerifiable.java [hunk 2]
```diff
@@ -174,7 +174,11 @@ public MethodBufferingJsonVerifiable isEqualTo(Number value) {
 		if (this.delegate.isAssertingAValueInArray() && containsAMatcher) {
 			readyToCheck.methodsBuffer.offer(".value()");
 		} else {
-			readyToCheck.appendMethodWithValue("isEqualTo", String.valueOf(value));
+			if(value instanceof Long) {
```
Comment (marcingrzejszczak) [inline spring-cloud-contract-verifier/src/main/groovy/org/springframework/cloud/contract/verifier/util/DelegatingJsonVerifiable.java]: WDYT about such a one liner instead?

```java
readyToCheck.appendMethodWithValue("isEqualTo",  value instanceof Long ? String.valueOf(value).concat("L") : String.valueOf(value))
```

**Comment by marcingrzejszczak** [comment]
Congrats @FelipeAdorno on your first PR :D Let's wait for the CI to build it but in the meantime there are 2 small remarks that I have.

**Pull request merged**